src/types/User.ts

```diff
@@ -0,0 +1,5 @@
+export interface User {
+  id: number,
+  name: string,
+  email: string,
```

Comment:
Consider using a more specific type for email, such as `Email` or `EmailAddress`, instead of `string`. 📧